tests/core/test_logging.py

```diff
@@ -116,6 +116,21 @@ def test_logging_error_tracebacks(caplog):
     assert "ValueError" in details["Fields"]["traceback"]
 
 
+def test_logging_exc_info_false(caplog):
+    """Ensure log formatter does not include exception traceback information
+    when exc_info is False"""
+    try:
+        raise ValueError("\n")
+    except Exception:
+        logging.exception("there was an error", exc_info=False)
+    details = assert_records(caplog.records)
+
+    assert details["Severity"] == 3
+    assert details["Fields"]["msg"] == "there was an error"
+    assert "error" not in details["Fields"]
```

Comment:
This is causing a test error in Python 2, it seems that `error` and `traceback` *are* present, I'm not sure why. Any ideas of what to do here ? I haven't been dealing with Python 2 for a while now...

Reply:
Curious, can you check with PDB what exc_info is on Python 2.7? Does it evaluate differently on 2.x?

Reply:
- On python 3.8, when `exc_info=False` is passed, `record.exc_info` is `False`.
- On python 2.7, when `exc_info=False` is passed... `record.exc_info` is a tuple with the exception and the traceback anyway...

Reply:
Well, that is certainly disappointing, so as long as dockerflow supports Python 2, we should probably extend the condition so the tests pass.

Reply:
Could I just skip this test in python 2 ?

Reply:
Hmmyeah, how about we have a test for each Python version?

Reply:
Yeah let's do that. c650711188da93470bda9820fb8b6d5cab178e2f